File: GFG/Array/Day1/MaxConsecutiveBit.java
public class MaxConsecutiveBit {
    public int maxConsecBits(int[] arr) {
        // code here
        if(arr.length==1){
            return 1;
        }
        
        int res,count;
        res=0;
        count=1;
        
        
        for(int i=1;i<arr.length;i++){
            if(arr[i]==arr[i-1]){
                count+=1;
            }
            else{
                count=1;
            }
            
            res=Math.max(res,count);
        }
        
        return res;
    }
}

File: GFG/Array/Day1/MaxConsecutiveBit.py
class Solution:
    def maxConsecBits(self, arr):
        #code here 
        if(len(arr)==1):
            return 1
        
        count=1
        res=1
        
        for i in range(len(arr)-1):
            if(arr[i+1]==arr[i]):
                count+=1
                res=max(count,res)
            else:
                count=1
                
        
        return res